<ui version="4.0" >
 <class>Frame_Visual</class>
 <widget class="QFrame" name="Frame_Visual" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Frame</string>
  </property>
  <property name="frameShape" >
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow" >
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QComboBox" name="comboBox" />
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2" />
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonShowAll" >
        <property name="text" >
         <string>Show All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonZoomOut" >
        <property name="text" >
         <string>Zoom Out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonZoomIn" >
        <property name="text" >
         <string>Zoom In</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView" >
     <property name="backgroundBrush" >
      <brush brushstyle="NoBrush" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="resizeAnchor" >
      <enum>QGraphicsView::AnchorViewCenter</enum>
     </property>
     <property name="viewportUpdateMode" >
      <enum>QGraphicsView::SmartViewportUpdate</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
